roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -55,6 +55,45 @@
   when:
     - youki_enabled
 
+- name: Cri-o | read default container runtime from crio.conf
+  ansible.builtin.shell: |
+    set -o pipefail && grep "^default_runtime" /etc/crio/crio.conf | awk -F'=' '{print $2}' | xargs
+  register: existing_crio_default_runtime
+  when: ansible_facts.services['crio.service'] is defined
+
+- name: Cri-o | set existing crio default runtime fact
+  ansible.builtin.set_fact:
+    existing_crio_default_runtime_output: "{{ existing_crio_default_runtime.stdout }}"
+  when: ansible_facts.services['crio.service'] is defined
```

Comment:
No need for set_fact when you already have a registered variable, just use it directly.